CFC_DataCollector/recommender/e_mission_trip_parsing.py

```diff
@@ -0,0 +1,15 @@
+from trips import E_Mission_Trip
+
+#main function, converting json into a list of E_Mission_Trips
+def get_trips_from_json_section():
```

Comment:
@joshzarrabi, @sdsingh, can you please resolve what needs to be here and what needs to be in `common`?